src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/SemanticModel/PXGraph/PXGraphEventSemanticModel.cs

```diff
@@ -199,6 +198,46 @@ private PXGraphEventSemanticModel(PXGraphSemanticModel baseGraphModel, Cancellat
 			return eventsGraphModels;
 		}
 
+		public IEnumerable<GraphEventInfoBase> GetAllEvents()
+		{
+			IEnumerable<GraphEventInfoBase>? allEvents = RowSelectingByName.Values;
+
+			AppendRowEvents(RowSelectedByName);
+			AppendRowEvents(RowInsertingByName);
+			AppendRowEvents(RowInsertedByName);
+			AppendRowEvents(RowUpdatingByName);
+			AppendRowEvents(RowUpdatedByName);
+			AppendRowEvents(RowDeletingByName);
+			AppendRowEvents(RowDeletedByName);
+			AppendRowEvents(RowPersistingByName);
+			AppendRowEvents(RowPersistedByName);
+
+			AppendFieldEvents(FieldSelectingByName);
+			AppendFieldEvents(FieldDefaultingByName);
+			AppendFieldEvents(FieldVerifyingByName);
+			AppendFieldEvents(FieldUpdatingByName);
+			AppendFieldEvents(FieldUpdatedByName);
+
+			AppendFieldEvents(CacheAttachedByName);
+			AppendFieldEvents(CommandPreparingByName);
+			AppendFieldEvents(ExceptionHandlingByName);
+
+			return allEvents;
+
+			//------------------------------------Local Function----------------------------------------------
```

Comment:
From my point of view if local functions are so hard to read without such comment, maybe it is worth to not use it at all.
Since you are the only active maintainer, probably it is ok to leave the code as is.